messaging/3.1/_index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: "Jakarta Messaging 3.1 (Under Development)"
-date: 2021-05-19
+title: "Jakarta Messaging 3.1"
+date: 2021-02-17
```

Comment:
Maybe needs to be updated to 2022 if possible

Reply:
I've updated this to 2022-03-15 which is in the release record: https://projects.eclipse.org/projects/ee4j.messaging/releases/3.1.0. it will probably need to be updated to match with the reality.